src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -18,6 +18,24 @@ public override FileInfo GetExistingItem()
             return new FileInfo(path);
         }
 
+        public FileInfo GetNonZeroMilliSec()
+        {
+            FileInfo fileinfo = new FileInfo(GetTestFilePath());
+            for (int i = 0; i < 5; i++)
+            {
+                fileinfo.Create().Dispose();
+                if (fileinfo.LastWriteTime.Millisecond != 0)
+                    break;
+
+                // This case should only happen 1/1000 times, unless the OS/Filesystem does
+                // not support millisecond granularity.
+
+                // If it's 1/1000, or low granularity, this may help:
+                Thread.Sleep(1234);
```

Comment:
And if the OS/FileSystem doesn't support millisecond granularity, isn't each call to this `GetNonZeroMilliSec` function going to take at least 5 seconds?  Do all OSes/FileSystems we test in CI support millisecond granularity / only call this method if they do?

Reply:
> Do all OSes/FileSystems we test in CI support millisecond granularity / only call this method if they do?

yes, if the Oses/FileSystem support millisecond granularity, then only we call this function